src/accountAPI.yaml

```
@@ -447,6 +455,7 @@ paths:
         - iso20022
       summary: Retrieve the designated ISO20022 XML camt.052 document
       description: Retrieve the designated ISO20022 XML camt.052 document.
+      operationId: retrieveIso20022AccountReports
```

```suggestion
      operationId: retrieveIso20022AccountReport
```
Use `Id` or not based on previous decision